services/production-tools/values-idfint.yaml

```diff
@@ -0,0 +1,13 @@
+production-tools:
```

Comment:
Since the chart is directly in Phalanx, you should delete this level and put the settings at the top level.